src/app/home/home-shared/validation-general-msg/validation-general-msg.component.html

```diff
@@ -0,0 +1,3 @@
+<p class="invalid-msg-container">
+  <span *ngIf="form.submitted && form.status === 'INVALID'" class="invalid-msg">Fields cannot be empty</span>
```

Comment:
Probably could be better to apply ngIf on container, instead of span. It's not necessery to have empty p tag on false.